src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -3273,6 +3273,7 @@ internal static bool IsSynthesizedAttributeOptional(WellKnownMember attributeMem
                 case WellKnownMember.System_STAThreadAttribute__ctor:
                 case WellKnownMember.System_Runtime_CompilerServices_AsyncStateMachineAttribute__ctor:
                 case WellKnownMember.System_Runtime_CompilerServices_IteratorStateMachineAttribute__ctor:
+                case WellKnownMember.System_Runtime_CompilerServices_TupleElementNamesAttribute__ctorTransformNames:
                     return true;
```

Comment:
No, actually System_Runtime_CompilerServices_TupleElementNamesAttribute__ctorTransformNames should not be added either. It is not generally optional

---
In reply to: [79713713](https://github.com/dotnet/roslyn/pull/13923#discussion_r79713713) [](ancestors = 79713713,79673650)